plugins/modules/scaleway_secret_version.py

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2023, Scaleway
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+import base64
+from scaleway_core.api import ScalewayException
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: scaleway_secret
+short_description: Manage Scaleway secret's secret version
+description:
+    - This module can be used to manage Scaleway secret's secret version.
+version_added: "2.1.0"
+author:
+    - Nathanael Demacon (@quantumsheep)
+extends_documentation_fragment:
+    - scaleway.scaleway.scaleway
+    - scaleway.scaleway.scaleway_waitable_resource
+requirements:
+    - scaleway >= 0.6.0
+options:
+    state:
+        description:
+            - Indicate desired state of the target.
+            - C(present) will create a new secret's version. If the secret does not exist, it will be created.
+            - C(absent) will delete the secret version, if it exists.
+            - C(disable) will disable the secret version, if it exists.
+            - C(enable) will enable the secret version, if it exists.
+            - C(access) will access the secret version, if it exists.
+        default: present
+        choices: ["present", "absent"]
+        type: str
+    secret_id:
+        description: secret_id
+        type: str
+        required: false
+    name:
+        description: name
+        type: str
+        required: true
```

Comment:
`name` is not present in the API

Reply:
You're right, unfortunatly there is an inconsistency in our api.
The create_secret function has a `name` argument while the get_secret_by_name has a `secret_name` one. We will fix this in our v1 api, both function will take a `name` argument.
I propose to keep `name` for both in ansible

Reply:
Ok, I'm fine with it! Thanks